content/docs-v1/site-deploy.md

```diff
@@ -93,30 +93,36 @@ Should you need to make any updates to your site, just merge the changes and Net
 
+{{< q-class "box warn" >}}
+- While the below instructions work as of December 2023, we have found that there are occasionally glitches. If you encounter any issues following the instructions below, please post about it on our [community forum](https://github.com/thegetty/quire/discussions) so we can provide assistance and improve our documentation. 
+{{< /q-class >}}
+
 ### Publish Your Project with Github Pages
 
 1. First, if you haven't already, follow the steps to host your project's code on GitHub in the [*GitHub*](/docs-v1/github) section of our documentation to create a repository for your project.
 
-2. Update the `url` in `content/_data/publication.yaml` to correspond to your own GitHub site. It will follow the pattern: https://YOUR-USERNAME.github.io/YOUR-PROJECT-DIRECTORY-NAME/. So, if your GitHub username is bonjovi and your project file is blaze-of-glory, your `baseURL` would be https://bonjovi.github.io/blaze-of-glory/. (Note: your GitHub username can also be your organization name if that is how your account is set-up in GitHub.)
+2. Update the `url` in `content/_data/publication.yaml` to correspond to your own GitHub site. It will follow the pattern: https://YOUR-USERNAME.github.io/YOUR-PROJECT-DIRECTORY-NAME/. So, if your GitHub username is bonjovi and your project file is blaze-of-glory, your `baseURL` would be https://bonjovi.github.io/blaze-of-glory/. (Note: your GitHub username can also be your organization name if that is how your account is set-up in GitHub.) Be sure to commit this update and push to origin. 
 
-3. In your command-line shell run `git checkout -b gh-deploy`, which creates a new temporary branch called `gh-deploy` to work in and switches you to that branch.
+3. In your command-line shell, make sure you are in your project folder and run `git checkout -b gh-deploy`, which creates a new temporary branch called `gh-deploy` to work in. This command will automatically switch you to that branch.
 
 4. Next run `quire build`.
 
 5. In your text editor, open the `_site` directory and add an empty file called `.nojekyll`, which lets GitHub know the site you're publishing [isn't a Jekyll site](https://github.blog/2009-12-29-bypassing-jekyll-on-github-pages/).
 
 6. Back in your command-line shell run `git add -f _site && git commit -m "Github pages deploy"`. This will add the contents of the `_site` directory to your project's git history so that it can be pushed to GitHub.
 
-7. If you have previously deployed the project to GitHub Pages, delete that old version by running `git push origin --delete gh-pages`. If this is your first deploy, skip this step.
+7. If you have previously deployed the project to GitHub Pages, delete that old version by running `git push origin --delete gh-pages`. **If this is your first deploy, skip this step.
```

Comment:
I think there's a missing set of asterisks here to make some of the text bold

Reply:
fixed!